docker-compose.yml

```diff
@@ -1,79 +1,82 @@
-trustroots:
-  container_name: trustroots
-  restart: always
-  build: .
-  # Mount the app dir in the container so our changes
-  #  to the app code are also changed in the container
-  #
-  # Mount node_modules separately as a data volume, so
-  #  that they won't get overwritten by app directory
-  #  Read more: http://stackoverflow.com/a/32785014/1984644
-  volumes:
-   - ./:/trustroots
-   - /trustroots/node_modules
-   - /trustroots/public
-   - /trustroots/uploads
-  command: npm run start:docker
-  links:
-    - mongodb:mongodb
-    - maildev:maildev
-    - influxdb:influxdb
-  ports:
-    - "3000:3000"
-    - "5858:5858"
-    - "35729:35729"
-  environment:
-    - NODE_ENV=development
-    - DB_1_PORT_27017_TCP_ADDR=mongodb
-    - PORT=3000
-    - DOMAIN=trustroots.dev
-    - VIRTUAL_HOST=localhost,trustroots.dev
-    - VIRTUAL_PORT=3000
+version: "3"
 
-## SMTP Server + Web Interface for viewing and testing emails during development.
-## http://djfarrelly.github.io/MailDev/
-maildev:
-  image: djfarrelly/maildev
-  container_name: maildev
-  restart: always
-  ports:
-    - "1080:1080"
+networks:
+  trustroots:
 
-## Database container
-mongodb:
-  image: mongo:3.6
-  container_name: db_1
-  restart: always
-  command: "--smallfiles --logpath=/dev/null"
-  ports:
-    - "27017:27017"
+volumes:
+  node-modules:
+  public:
+  uploads:
 
-## InfluxDB container
-## https://github.com/tutumcloud/influxdb
-influxdb:
-  image: tutum/influxdb:latest
-  container_name: influxdb
-  restart: always
-  ports:
-    - "8083:8083"
-    - "8086:8086"
-  environment:
-    - ADMIN_USER=root
-    - INFLUXDB_INIT_PWD=root
-    - PRE_CREATE_DB=trustroots
+services:
+  trustroots:
+    container_name: trustroots
+    restart: always
+    build: .
+    # Mount the app dir in the container so our changes
+    #  to the app code are also changed in the container
+    #
+    # Mount node_modules separately as a data volume, so
+    #  that they won't get overwritten by app directory
+    #  Read more: http://stackoverflow.com/a/32785014/1984644
+    networks:
+      - trustroots
+    volumes:
+      - ./:/trustroots
+      - node-modules:/trustroots/node_modules
+      - public:/trustroots/public
+      - uploads:/trustroots/uploads
+    command: npm run start
+    depends_on:
+      - mongodb
+      - maildev
+      - influxdb
+    ports:
+      - "3000:3000"
+      - "5858:5858"
+      - "35729:35729"
```

Comment:
This can go away, it was for livereload which we don't use any more (webpack dev server connects via websocket connection on the same port for the main service, 3000).

Reply:
Well we gotta keep 3000 so we can access it from browser

Reply:
Yup, I just meant port `35729` :)